src/styles/components/_commentItem.scss

```diff
@@ -142,4 +146,19 @@
   h3 {
     font-size: 1.4rem;
   }
-}
\ No newline at end of file
+}
+
+.edit {
+  display: flex;
+  justify-content: flex-end;
+  .span {
+    margin-right: 20px;
+  }
+  .mr-4.edit-button {
+    margin-left: unset !important;
+    margin-right: unset !important;
+  }
+  .jXLEy {
```

Comment:
What classname is this?

Reply:
It's from the styled button component, I had to target it to align the cancel and submit button

Reply:
I'm aware it's from a styled component. My point is, doesn't that classname change? If yes, then you'd need to add your own static classname and style it.

Reply:
The classname does not change